Doc/library/collections.rst

```diff
@@ -1186,7 +1189,11 @@ anywhere a regular dictionary is used.
    method.
 
 .. versionchanged:: 3.9
-    Added merge (``|``) and update (``|=``) operators, specified in :pep:`584`.
+   Added merge (``|``) and update (``|=``) operators, specified in :pep:`584`.
+
+.. versionchanged:: 3.14
```

Comment:
Now I'm wondering how to say it here if the Python implementation does not raise such error =/ (it's still important to say that the exception is raised I think)